src/load.ts

```diff
@@ -78,6 +78,54 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param endpoint  The URL to the CDN.
+ * @param options  Optional parameters.
+ */
+export async function loadCdn(endpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;

```

Comment:
I'm wondering if it makes more sense to name it `loadFromCDN` insead.

Reply:
+1

Reply:
I will use camelCase, so it will be `loadFromCdn`